demos/rag_agentic/notebooks/Level4_agentic_and_mcp.ipynb

```diff
@@ -537,6 +551,18 @@
     "https://www.ibm.com/granite/docs/models/granite/#function-calling and observed that it produced more reliable results"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "db50c65c-fd58-4655-917b-7fb34574dba2",
+   "metadata": {},
+   "source": [
+    "We will also be using the [Slack MCP Server](https://github.com/modelcontextprotocol/servers/tree/main/src/slack) for this query. If you haven't already, you can follow the instructions [here]() to install the Slack MCP server. Once you have the Slack MCP server running, you will need to:\n",
+    "\n",
+    "- Setup a slack app with your Slack workspace by following the steps [here](https://github.com/modelcontextprotocol/servers/tree/main/src/slack#setup)\n",
```

Comment:
Should this link to the setup instructions in our repo that are OCP specific? https://github.com/opendatahub-io/llama-stack-demos/tree/026f0f72fa86e2709e66e753be4a98ba2f15b5b9/mcp-servers/slack#setting-up-on-ocp

Reply:
ah yes, will update that